client/.env

```diff
@@ -1,2 +0,0 @@
-ESLINT_NO_DEV_ERRORS=true
-token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MTYsIm5hbWUiOiJramJkaiIsImlzQWRtaW4iOmZhbHNlLCJpYXQiOjE2MzU3MDUyNTN9.a95ckExrE6Msrn0iRmGMfl5KTId4nDuqAh8BkoQqa2c
```

Comment:
I don't know how did I miss this in my previous reviews, but it is incredibly **DANGEROUS** to push access tokens to git, if this was a real project, you should immediately change this token and not treat this lightly.

Reply:
I suppose this is a Inadvertent mistake, we will use a new token.